test/extended/etcd/leader_changes.go

```diff
@@ -17,14 +18,24 @@ var _ = g.Describe("[sig-etcd] etcd", func() {
 	defer g.GinkgoRecover()
 	oc := exutil.NewCLIWithoutNamespace("etcd-leader-change").AsAdmin()
 	g.It("leader changes are not excessive [Late]", func() {
+		controlPlaneTopology, err := exutil.GetControlPlaneTopology(oc)
+		o.Expect(err).NotTo(o.HaveOccurred())
+		if *controlPlaneTopology == configv1.ExternalTopologyMode {
```

Comment:
should you exclude the case that platform is IBM Cloud ?

Reply:
No idea, they shouldn't have that metric inside the guest cluster either so it will just fail differently there now or not?

Reply:
@csrwng does that address the concern?

Reply:
Yes, let's just leave it as is and if needed, it can be updated later to accomodate ROKS